docs/deploy-rbd.md

```diff
@@ -80,6 +81,11 @@ Admin credentials are required for provisioning new RBD images `ADMIN_NAME`:
 `ADMIN_PASSWORD` - note that the key of the key-value pair is the name of the
 client with admin privileges, and the value is its password
 
+If clusterID is specified, then a pair of secrets are required, with keys named
+`subjectid` and `credentials`. Where, `subjectid` is the name of the client
+with admin privileges and `credentials` contain its password. The pair required
```

Comment:
Ceph seems to use id and key as the CLI args for the same. I used credentials instead of keys, as it could be any form of credentials. Also, **if** we have different creds from the provisioner and the publisher, then wanted to call the "user" the "subject" instead. This was my reasoning.

I would rather move these to Ceph common terminology, so that it is easily relatable to a Ceph user wanting to administer the CSI as well. Would that be `id` and `key` hence? @rootfs thoughts?

Reply:
Updated in latest commit. We are now back to regular programming with user/admin id/key, although for RBD it does not seem to be required, and should be the same IMO. Something to take up at a later time.

Reply:
@ShyamsundarR the docs still seem to use `subjectid` and `credentials`